sycl/test/basic_tests/Intel_get_backend.cpp

```diff
@@ -0,0 +1,55 @@
+// RUN: %clangxx -fsycl %s -o %t.out
```

Comment:
The file should not be named "Intel_...".

Reply:
Fixed.